build_thirdparty.sh

```diff
@@ -39,3 +39,32 @@ set -x
 
 # shellcheck disable=SC2086
 python3 "$YB_THIRDPARTY_DIR/python/yugabyte_db_thirdparty/yb_build_thirdparty_main.py" "$@"
+
+rc=$?
```

Comment:
The vast majority of our shell scripts run in the `set -euo pipefail` mode so that we don't have to check return codes after every commad. For this reason, in particular, grabbing the error code here is useless, because it cannot be different than zero. The script would have terminated in that case.